.github/workflows/infection.yml

```diff
@@ -0,0 +1,36 @@
+name: Infection Test
+
+on:
+  schedule:
+    - cron: '33 3 * * 1' # weekly, on Monday morning
+
+env:
+  MAILER_DSN: null://null
+  ILIOS_LOCALE: en
+  ILIOS_SECRET: ThisTokenIsNotSoSecretChangeIt
+  ILIOS_FILE_SYSTEM_STORAGE_PATH: /tmp
+  SYMFONY_DEPRECATIONS_HELPER: disabled=1
+  MESSENGER_TRANSPORT_DSN: doctrine://default
+  latest_php: 8.1
+  DOCKER_BUILDKIT: 1
+jobs:
+  infection:
+    name: Infection tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use PHP ${{ env.latest_php }}
+        uses: shivammathur/setup-php@2.18.1
+        with:
+          coverage: none
+          php-version: ${{ env.latest_php }}
+          extensions: apcu
+      - name: install dependencies
+        run: composer install --no-interaction --prefer-dist
+      - name: create coverage
+        run:
+          php -d xdebug.mode=coverage vendor/bin/phpunit --coverage-xml=build/coverage --log-junit=build/coverage/phpunit.junit.xml
+      - name: infection
```

Comment:
Can you use `actions/upload-artifact` and save the result HTML? Then it will end up at the bottom of each action run like https://github.com/ilios/ilios/actions/runs/2320799106 so we can view it.